x-pack/plugins/search_inference_endpoints/public/types.ts

```diff
@@ -11,6 +11,8 @@ import { AppMountParameters } from '@kbn/core/public';
 import { MlPluginStart } from '@kbn/ml-plugin/public';
 import { SharePluginStart } from '@kbn/share-plugin/public';
 import React from 'react';
+
+import { ServerlessPluginStart } from '@kbn/serverless/public';
```

Comment:
Ideally this would be

```suggestion
import type { ServerlessPluginStart } from '@kbn/serverless/public';
```

but we're not doing that for all the other imports so non-blocking